src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./Footer.css";
+import emailIcon from "../../Images/emailIcon.png";
+
+export default function Footer() {
+    return (
+        <footer>
+            <section id="links">
+                <ul className="icons">
+                    <li>
+                        <a
+                            href="https://github.com/kfoley123"
```

Comment:
you can make these web addresses into variables so they are easier to change in the future if need be, but also you really only use them here so no big deal